runtime/lua/vim/lsp/util.lua

```diff
@@ -1226,6 +1229,7 @@ function M.jump_to_location(location, offset_encoding, reuse_win)
       'jump_to_location must be called with valid offset encoding',
       vim.log.levels.WARN
     )
+    offset_encoding = vim.lsp.get_active_clients({ bufnr = 0 })[1].offset_encoding
```

Comment:
Same here, why the behavior change?

Reply:
> because they are too small just one line or two lines. it worth on a separator pr ?

Yes it's much easier to review

To unblock this PR: Could you revert all offset_encoding related changes in this PR? These need a closer look.  The rest looks good

Reply:
Okay and i see the `ctx.bufnr` you fix in another pr. need rebase then

Reply:
@mfussenegger  LGTY?